packages/genesys-spark-components/src/components/stable/gux-popover-list/gux-popover-list.tsx

```diff
@@ -205,6 +205,8 @@ export class GuxPopoverList {
           'gux-hidden': !this.isOpen,
           'gux-popover-wrapper': true
         }}
+        role="dialog"
+        aria-describedby="gux-popover-content"
```

Comment:
I am not sure about the dialog role for a popover. Did you have a reference for using this role for a popover?

Reply:
It's a good point. I see in our spark repo that we use gux-popover more like a dialog since the user clicking on an element triggers the popover to show. However, do clients using spark use the popover more like a tooltip where it shows on hover?

Reply:
It's important to keep in mind that this ticket was created `December 15, 2021 at 3:49 PM`. A lot has changed since then and what's written in these older tickets and comments is not necessarily always still relevant. 

The three issues documented in the ticket were:
_1. Content in popover is only accessible on mouse hover_

The example page should probably be updated to show the popover showing on button press instead of hover. The [Spark Popover documentation](https://spark.genesys.com/latest/components/feedback/popovers/usage-wA2Lqhzx) now states that 'popovers are triggered on click and persistent'. We should update the example page to highlight this use case.

_2. Consider using `role="tooltip"` for popover_

This ticket was written before the `dialog` role was available to use in all browsers. I don't believe the `tooltip` role is usable here, since the design system now suggests that the popover should be triggered "on click".  Tooltips are supposed to be triggered on hover, generally. Since `role=tooltip` is probably not appropriate, we need to do more research to confirm whether or not `dialog` is an appropriate role for this component. 

_3. Consider trying `aria-describedby` to link popover target and popover target_ 

This is trying to say that the **target** (button or something that opens the popover) should have the aria-describedby attribute linking to the popover content (id on the content)

Reply:
@katie-bobbe-genesys Correct me if I'm wrong, but will the client need to remember to add `aria-describeby` to the target element that opens the popover? I could add it to the target element in the popover's example page but wanted to clarify this.